hipogram/posts/admin.py

```diff
@@ -1,5 +1,18 @@
 from django.contrib import admin
+from .models import Post, Tag
 
-from .models import Post
+admin.site.register(Tag)
 
-admin.site.register(Post)
+
+class PostAdmin(admin.ModelAdmin):
+    list_display = ("created_by", "creation_datetime", "Tags")
```

Comment:
`created_by` works but I think you want it to appear as `username`.
How should I use that method in admin.py?

Reply:
I tested this now and it already shows `username`. It seems [Django's User model's __str__](https://github.com/django/django/blob/main/django/contrib/auth/base_user.py#L64) returns `username`. No need a change here. But still, it would be nice to learn how to use a model's `__str__` method in `list_display` field. It's not a tricky thing, don't overthink about this.